project/Dependencies.scala

```diff
@@ -67,7 +65,8 @@ object Dependencies {
     "com.typesafe.akka" %% "akka-http-jackson"    % AkkaHttp,
     "com.typesafe.akka" %% "akka-http-spray-json" % AkkaHttp,
     "com.typesafe.akka" %% "akka-http-testkit"    % AkkaHttp,
-    "com.typesafe.akka" %% "akka-http-xml"        % AkkaHttp
+    "com.typesafe.akka" %% "akka-http-xml"        % AkkaHttp,
+    "com.typesafe.akka" %% "akka-http2-support"   % AkkaHttp
```

Comment:
Since 10.2.3 the `akka-http2-support` artifact is noop and HTTP/2 support is included in the main `akka-http` artifact. We can remove `akka-http2-support` from the BOM

Reply:
In the guide example something else was bringing `akka-http2-support` `10.2.3` onto the class path causing an initialization failure (incompatible versions) when the BOM was bringing in `akka-http` `10.2.4`.  Maybe it was because `akka-grpc` doesn't have the latest version of `akka-http` yet?

Reply:
Aaaah, you're right!

Until `akka-http2-support` is no longer brought in transitively we must keep it around.